apps/backend/src/lib/schema/agri/constants.ts

```diff
@@ -0,0 +1,64 @@
+export const VERSION = "1.2.0";
+
+export const DOMAIN = ["ONDC:AGR11","ONDC:AGR10"];
+
+export const SRV_FULFILLMENT_TYPE = ["Home-Service", "Store-Service","Seller-Fulfilled", "Buyer-Fulfilled"];
+
+export const SRV_INTENT_TAGS = ["finder_fee_type","finder_fee_amount","FINDER_FEE_TYPE","FINDER_FEE_AMOUNT","BAP_TYPE","BAP_Terms","BAP_TERMS","BAP_DETAILS","BUYER_ID_CODE","BUYER_ID_NO","BUYER_ID"];
+
+export const SRV_PAYMENT_TYPE = [
+	"PRE-FULFILLMENT",
+	"ON-FULFILLMENT",
+	"POST-FULFILLMENT",
+];
+
+export const SRV_FULFILLMENT_STATE = [
+	"IN_TRANSIT",
+	"AT_LOCATION",
+	"COLLECTED_BY_AGENT",
+	"RECEIVED_AT_LAB",
+	"TEST_COMPLETED",
+	"REPORT_GENERATED",
+	"REPORT_SHARED",
+	"COMPLETED",
+	"Placed",
+	"cancel"
+];
+
+export const SRV_ORDER_STATE = [
+	"Created",
+	"Accepted",
+	"In-progress",
+	"Completed",
+	"Cancelled",
+	"COMPLETED",
+	"Pending",
+];
```

Comment:
While we are supposed to use .toUpperCase(), please try to use the upper case values only and keep the upper case consistent